.github/workflows/publish.yml

```diff
@@ -6,7 +6,7 @@ on:
   push:
     branches:
       - main
-  pull_request_target:
+  pull_request:
```

Comment:
it won't have access to secrets if you make this change.

Reply:
but only from 3rd party forks right? the build is designed to gracefully degrade without the secret so it just wont pull in some github specific data for the preview deployment